src/chainstate/stacks/transaction.rs

```diff
@@ -1637,7 +1861,7 @@ mod test {
                 version: 1,
                 bytes: Hash160([0xff; 20]),
             }
-            .into(),
+                .into(),
```

Comment:
I see a lot of changes like this that are just indentation. Are you using a different `rustfmt` config or did the defaults change?